samcli/commands/validate/validate.py

```diff
@@ -3,9 +3,11 @@
 """
 import os
 
+import logging
 import boto3
 from botocore.exceptions import NoCredentialsError
 import click
+import cfnlint.core # type: ignore
```

Comment:
The number of imports at the top level should be as small as possible, if we really need it we should move it within `cli` method. Also what is the reason behind the `#type: ignore` , I'd rather not have us skip any lint checks.

Reply:
Have moved below to method below. It seems cfn-lint does not have a py.typed file/type hints and we likely will not be adding them before release. Is it alright to suppress for now?